docs/examples/amazon-dynamodb-sessions.md

```diff
@@ -0,0 +1,92 @@
+Griptape provides [Conversation Memory](../griptape-framework/structures/conversation-memory.md) as a means of persisting conversation context across multiple Structure runs.
+If you provide it with a suitable a Driver, the memory of the previous conversation can be preserved between run of a Structure, giving it additional context for how to respond.
```

Comment:
"with a suitable a Driver" -> "with a suitable Driver"

(maybe were you playing Mario?)

Reply:
It's-a-fixed